ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2095,6 +2095,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: baremetal-sno-agent-ipv4-static-connected-p1-f14
```

Comment:
Missing `E2E_RUN_TAGS` for this job

Reply:
Thanks, I should also use baremetal-upi tag and specify arm64/amd64,  will fix them together in the next commit.